libraries/logging/stream-logger-android/build.gradle

```diff
@@ -0,0 +1,77 @@
+import io.getstream.chat.android.Dependencies
+import io.getstream.chat.android.Configuration
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'de.mannodermaus.android-junit5'
+
+ext {
+    PUBLISH_GROUP_ID = Configuration.artifactGroup
+    PUBLISH_ARTIFACT_ID = 'stream-logger-android'
+    PUBLISH_VERSION = rootVersionName
+}
+
+apply from: "${rootDir}/scripts/publish-module.gradle"
```

Comment:
Do we want to publish those libraries?

Reply:
I think we will need it as well we need it on the `core` module.
It won't be a "direct" dependency our customers will use, but a transitive dependency that could be needed